scripts/helpers/getlistjson.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
```

Comment:
Is this still being used? I think this can be removed.

Reply:
It's in all other JS files as well.